scripts/update_snapshots.sh

```diff
@@ -6,16 +6,33 @@ set -eu
 toplevel="$(git -C "$(dirname "$0")" rev-parse --show-toplevel)"
 cd "${toplevel}"
 
-yarn backend
-echo "Updating GitHub graphql flow types"
-node ./bin/generateGithubGraphqlFlowTypes.js > src/plugins/github/graphqlTypes.js
-echo "Updating for sharness/test_load_example_github.t"
+tmpdir="$(mktemp -d)"
+cleanup() {
+    rm -r "${tmpdir}"
+}
+trap cleanup EXIT
+
+SOURCECRED_BIN="${tmpdir}/bin"
+yarn run --silent backend --output-path "${SOURCECRED_BIN}"
+export SOURCECRED_BIN  # for Sharness and shell tests
+export NODE_PATH="${toplevel}/node_modules${NODE_PATH:+:${NODE_PATH}}"
+
+echo "Updating GitHub GraphQL Flow types"
+cp .prettierrc.json "${SOURCECRED_BIN}/"
+node "${SOURCECRED_BIN}/generateGithubGraphqlFlowTypes.js" \
+    >src/plugins/github/graphqlTypes.js
+
+echo "Updating sharness/test_load_example_github.t"
 (cd sharness; UPDATE_SNAPSHOT=1 ./test_load_example_github.t -l)
+
 echo "Updating git/loadRepositoryTest.sh"
-./src/plugins/git/loadRepositoryTest.sh -u
+./src/plugins/git/loadRepositoryTest.sh -u --no-build
```

Comment:
Good catch.

Reply:
I’d actually assumed that this was intentional—you omit `--no-build` on
the first one so that it builds it into the main repository, then the
rest depend on that. The unpleasant asymmetry is what first prompted me
to write the PR. :-)

Reply:
I always had a `yarn backend` call at the top. I think I just forgot to pass --no-build to the first one.